### ISSUE TITLE
SSE does not work again... 

### ISSUE THREAD

**Issue by damianpumar** [honojs/hono#3765]
### What version of Hono are you using?

latest

### What runtime/platform is your app running on? (with version if possible)

node

### What steps can reproduce the bug?

The sse does not work again in the latest version...
I feel super frustrated after spending 1 hour and not understanding what was happening with my code, when I moved to the version "@hono/node-server": "1.8.2" and  "hono": "4.1.1", it works correctly, I have already fixed this same error in the past here https://github.com/honojs/hono/commit/b827efb3fb3fa988e74a5b96d1cfcd75d415f2af

### What is the expected behavior?

SSE should works

### What do you see instead?

-

### Additional information

-

**Comment by yusukebe** [comment]
Hi @damianpumar 

Can you fix it? I can't understand that behavior only with your information.

**Comment by damianpumar** [comment]
Yes, I can fix it again, but it was fixed by me some month ago... 
Basically, using the latest version, the SSE server does not work...

**Comment by yusukebe** [comment]
This issue is not described well and reproduced. We can close this.
